.github/container/Dockerfile.jax

```diff
@@ -54,14 +56,24 @@ RUN build-jax.sh \
     --xla-arm64-patch /opt/xla-arm64-neon.patch \ 
     --clean
 
+## Transformer engine: check out source and build wheel
+RUN <<"EOF" bash -ex -o pipefail
+pip install ninja && rm -rf ~/.cache/pip
+# TransformerEngine now needs JAX at build time
+pip install -e ${SRC_PATH_JAX}
```

Comment:
This line can actually be deleted because `build-jax.sh` does install JAX inside this `builder` stage.